packages/graphql/src/schema/resolvers/wrapper.ts

```diff
@@ -89,13 +90,21 @@ export const wrapResolver =
         context.callbacks = config.callbacks;
 
         if (!context.jwt) {
-            if (context.plugins.auth) {
-                // Here we will try to compute the generic Secret or the generic jwksEndpoint
-                const contextRequest = context.req || context.request;
-                context.plugins.auth.tryToResolveKeys(context instanceof IncomingMessage ? context : contextRequest);
+            if (authorization) {
+                const jwt = await new Neo4jGraphQLAuthorization(authorization).parseFrom({ context });
+                jwt && (context.jwt = jwt);
```

Comment:
I'm not too fond of this syntax tbh, but in any case, do we really need this?

`context.jwt` is undefined anyway if we reach this point so we could do `context.jwt=jwt` anyway right?